### PR TITLE
hwdb: various fixes to mm kbd hwdb files

### PR DESCRIPTION
**Pull request by poettering**
These are the last 4 commits from #34325. I split them out, as @martinpitt suggested we could just merge this bits already.

Hence, here it is.

@martinpitt ptal!

### CODE REVIEW

File: hwdb.d/60-keyboard.hwdb [hunk 1]
```diff
@@ -190,7 +190,7 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMateB311RN-31*:pvr*
 
 # Travelmate C300
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnTravelMate*C3[01]0*:*
- KEYBOARD_KEY_67=f24                                    # FIXME: rotate screen
+ KEYBOARD_KEY_67=rotate_display                         # rotate screen
```
Comment (YHNdnzj) [inline hwdb.d/60-keyboard.hwdb:193]: Hmm, as mentioned in the other PR, this is now self-descriptive and thus the comments are no longer necessary?

Comment (poettering) [reply hwdb.d/60-keyboard.hwdb:193]: dunno, wanted to make the minimum changes I could get away with. i.e. just dropped the FIXME. I mean, so many other comments in that file are kinda redundant. Let's maybe clean that up another day?

**Review by martinpitt — APPROVED** [review]
Thanks, LGTM! (Not a fan of the header inputs, but I trust they are a necessary evil)

The test failures seem unrelated (semaphore running out of disk, TEST-55-OOMD failure in TF, and TEST-64-UDEV-STORAGE-simultaneous_events/TEST-85-NETWORK-NetworkdSRIOVTests failures in Ubuntu CI)

**Comment by poettering** [comment]
thanks for the review!

**Pull request merged**